tarea1-Yefer.Garcia/assets/style/style.css

```diff
@@ -0,0 +1,66 @@
+@import url("https://fonts.googleapis.com/css2?family=Oswald:wght@500&family=Roboto:wght@500&display=swap");
+
+/* Centrado */
+* {
+  margin: 0;
+  box-sizing: border-box;
+}
+body {
+  display: flex;
+  justify-content: center;
+}
+.revista {
+  margin-top: 50px;
+  width: 600px;
+  height: 900px;
+  background-image: url(/tarea1-Yefer.Garcia/assets/img/shakira\ 2.jpg);
+  background-repeat: no-repeat;
+  background-size: 600px;
+  display: flex;
+  justify-content: space-between;
+  flex-direction: column;
+  align-items: center;
+}
+/* firts part */
+
+.letter-1-2 {
+  display: flex;
+  justify-content: flex-end;
+}
+
+.letter-1 h1 {
+  font-size: 120px;
+  color: rgb(255, 77, 195);
+  font-family: "Roboto", sans-serif;
+}
+
+h3 {
```

Comment:
please move all the element rules to the top, this one could be right after _body_ rule